Scala/syntax_test_scala.scala

```diff
@@ -306,6 +306,10 @@ type Foo = Bar[A] forSome { type A }
 // <- comment.block.documentation.scala
 */
 
+  /**/0xff
+//^^^^ comment.block.empty.scala
+//    ^^^^ - thereafter, not a comment 
```

Comment:
It demonstrates the comment has terminated and text thereafter is not regarded as a comment.  How does describing the hexadecimal number as a comment help in this case?

Kind regards